fv3core/pace/fv3core/stencils/d2a2c_vect.py

```diff
@@ -583,12 +616,13 @@ def __call__(self, uc, vc, u, v, ua, va, utc, vtc):
         )
 
         # tmp edges
-        self._avg_box(
-            u,
-            v,
-            self._utmp,
-            self._vtmp,
-        )
+        if self._grid_type < 3:
```

Comment:
I wonder if it wouldn't be more readable if we move the if out of here and have something in init stubbing those out. E.g.

```
def _avg_box_passthrough(...):
  pass

if self._grid_type < 3:
    self._avg_box = _avg_box_passthrough
```

Reply:
I think it's better to have the call function be clear about what's happening. If there are significant performance implications maybe?

Reply:
No significant perf implication at our scale